### ISSUE TITLE
Make mobile-install on windows

### ISSUE THREAD

**Issue by Oblivivious** [master-hax/pixel-backup-gang#2]
Hi,
I have downloaded make using choco on windows. Then running 'make mobile-install' gives me:
tar: Option --transform=s,^scripts/,pixel-backup-gang-snapshot/, is not supported
Usage:
  List:    tar -tf <archive-filename>
  Extract: tar -xf <archive-filename>
  Create:  tar -cf <archive-filename> [filenames...]
  Help:    tar --help
make: *** [Makefile:42: pixel-backup-gang-snapshot.tar.gz] Error 1

(Could this be a limitation of the tar command on windows, no '--transform' option?)

I also have no idea on how to do the process on the pixel phone. I'm just really new to this, and probably need a step by step process. I just find the current process a bit difficult to wrap my head around. Especially where to copy the scripts and files, what I need and where to run the files. Like an ELI5 would be amazing for amateurs like me who want to get the most out of my pixel phone.

Thanks for the help

**Comment by master-hax** [comment]
the makefile only works on linux. you can instead manually copy the scripts to the device. the easiest way to do that is to download the latest [release](https://github.com/master-hax/pixel-backup-gang/releases)

**Comment by master-hax** [comment]
i just added a new install script. you can just run the following from the pixel itself:

```
sh -c "$(curl -fSs https://raw.githubusercontent.com/master-hax/pixel-backup-gang/master/install.sh)"
```

**Comment by Oblivivious** [comment]
Thanks for simplifying the process,
Do I have to separately find and mount the ssd,
or is there something I can do after 
'''pixel backup gang successfully installed to /data/data/com.termux/files/home/pixel-backup-gang.'''

Im assuming it doesn't auto mount the ssd card as the os is still asking me to format the drive. In termux when I use:
bash show_devices.sh
It doesn't do anything for me  (I was in the downloads directory after opening the shell script from file manager and the file was saved in ~/downloads/)

Thank you

**Comment by Oblivivious** [comment]
I have installed many packages and in magisk, change mount namespace mode to global. and pasted all of the scripts to /data/data/com.termux/files/home/ as well as /data/data/downloads.

In termux I input:
su
./mount_ext4 ___
I get not running in global mount namespace, try elevating first

But the alternative:
su
./start_global_shell.sh
./mount_ext4.sh ____ or bash mount_ext4.sh
I get inaccessible or not found

If I do it from ~/downloads/ then nothing happens with all of the commands

**Comment by Oblivivious** [comment]
This is pretty much what I did, I tried way more things so I don't know if that changed anything (I didn't document them as it didn't go anywhere).
Mind the layout, I tried my best, and it was reordered to group actions together for simplification, in contrast, my trial and error method had a much more sporadic ordering.
Let me know if I did anything wrong, or if you'd change anything.

______Setup
-Use MiniTool Partition Wizard to format the external drive to ext4 on Windows
-Install the repo onto the pixel phone
-Needs Magisk
-Download Termux from f-droid
-in an adb folder open up cmd
-In Termux type 'su' to get root permission from Magisk
-Go to settings (pixel phone) and in permissions give access to storage

-Type:
termux-setup-storage           ***Dunno what this did tho but I did it so...

***Stuffs I installed
pkg install util-linux
pkg install tsu
pkg install blk-utils

***Switch repos         ***if you are having problems installing the packages
termux-change-repo  


_______Use to find external drive easier (By looking at file size)(use a new session (you can use two fingers to swipe from the left side)):
tsu
lsblk

***Look for the size of the drive/partition that closely matches your external drive (mine was sdg1) sdg should be at the very bottom

***The Block_device looks like:    /dev/block/sdg1
(To me it looks more like a path) (Correct me if I'm wrong, it just seemed to work for me) (This took so long to figure out, ultimately it was luck and desparation, I tried literally everything.)
__________________________________________________________________

_______Download the files
su
sh -c "$(curl -fSs https://raw.githubusercontent.com/master-hax/pixel-backup-gang/master/install.sh)"

_____________mount external drive
su
cd /data/data/com.termux/files/home/downloads
./start_global_shell.sh
cd /data/data/com.termux/files/home/downloads
./mount_ex4.sh /dev/block/sdg1

***Then I'm assuming that the the_binding folder is what I use to backup to and from. 
Would like clarification if there are any other indications that this worked. As I cannot see any other indication and the notification "Issue with drive, tap to fix" is still there.

**Comment by master-hax** [comment]
you don't need to do any of that extra stuff...you don't even need termux. just run the scripts as root.

yes, `/the_binding` on the internal "sd card" is mirrored to the one on the external drive

**Comment by master-hax** [comment]
Everything before `su` is unnecessary. All the steps after `su` look correct to me.
